src/Classes/ServiceAPI/MyRadio_TracklistItem.php

```diff
@@ -227,9 +231,7 @@ private static function trackAmalgamator($result, $playlists = true)
 
             if ($playlists) {
                 $playlistobjs = iTones_Playlist::getPlaylistsWithTrack($trackobj);
-                foreach ($playlistobjs as $playlist) {
-                    $track['in_playlists'] .= $playlist->getTitle() . ', ';
-                }
+                $track['in_playlists'] = implode(', ', array_map(function($i){return $i->getTitle();}, $playlistobjs);
```

Comment:
Parse error: syntax error, unexpected ';' in src/Classes/ServiceAPI/MyRadio_TracklistItem.php on line 234